src/api/database/models/User.ts

```diff
@@ -69,6 +74,30 @@ class User extends ParanoidModel<
 
   @HasMany(() => Interview)
   interviews: NonAttribute<Interview[]>;
+  
+
+
+
+  @BeforeDestroy
+  static async cascadeSoftDelete(user: User, options: any) {
+
+    const moment = require('moment-timezone');
+    const timestamp = moment().format('YYYY-MM-DD HH:mm:ss.SSSZ');
+
+    await GroupUser.update(
```

Comment:
I don't think it's the right thing to do. @BeforeDestroy needs to support both soft and hard deletion.